src/mokka/src/main/java/pl/hycom/mokka/service/file/DefaultFileService.java

```diff
@@ -0,0 +1,47 @@
+package pl.hycom.mokka.service.file;
+
+import org.apache.commons.lang.StringUtils;
+import org.apache.log4j.Logger;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.stereotype.Component;
+
+import java.io.File;
+
+/**
+ * @author Mariusz Krysztofowicz (mariusz.krysztofowicz@hycom.pl)
+ */
+@PropertySource("classpath:application.properties")
```

Comment:
I do not think we need to define PropertySource here as it already done (transparently) by Spring Boot